packages/documentation/src/pages/platform/tools/downtime-notifications.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Va.gov downtime notifications
+tags: pagerduty,maintenance windows,scheduled downtime,banner
```

Comment:
@jbalboni @rianfowler  Should all of these tags pull up `downtime notifications` in the search bar? It looks like `maintenance`, `scheduled`, and `banner` don't pull it up.

Reply:
Didn't dive into it but I think the search tool we're using might be looking for `, ` when breaking up phrases for indexing. searching for "window" works but not "maintenance"

Reply:
Could also be a bug with the display logic.